### PR TITLE
Vo overview statistics

### PR DESCRIPTION
**Pull request by bodnara**
* added table with members count per status to vo overview

### CODE REVIEW

**Review by HejdaJakub — COMMENTED** [review]
Implementation looks good, so I have only one comment. We will want to implement similar statistics also in group section further and maybe on a separate page, so I recommend to create new reusable component for this statistics.

**Review by HejdaJakub — APPROVED** [review]

**Comment by Vojtech-Sassmann** [comment]
Hi @bodnara, if there are no more changes requested, please rebase this PR. You can even merge it and update the issue status in Redmine (move it to resolved and write changelog to wiki)

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-overview/vo-overview.component.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <mat-spinner *ngIf="loading" class="mr-auto ml-auto"></mat-spinner>
 <div *ngIf="!loading">
+  <app-statistics-card
+    [rowNames]="rowNames"
+    [statistics]="membersCount"
+    [title]="'VO_DETAIL.OVERVIEW.MEMBERS_COUNT'">
+  </app-statistics-card>
```
Comment (xkureck) [inline apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-overview/vo-overview.component.html:7]: I suggest moving the statisticts under the perun-web-apps-menu-buttons-field
I dont think that the statistics are that important so they are at the top, the box button are more important in my opinion

**Review by Vojtech-Sassmann — CHANGES_REQUESTED** [review]
Please, align the card with the rest of the content. You can do it just by setting the left margin to 0.

**Review by Vojtech-Sassmann — APPROVED** [review]

**Pull request merged**